akka-typed-testkit/src/main/scala/akka/typed/testkit/Effects.scala

```diff
@@ -20,7 +20,7 @@ import scala.concurrent.duration.{ Duration, FiniteDuration }
 abstract class Effect
 
 object Effect {
-  @SerialVersionUID(1L) final case class Spawned(childName: String) extends Effect
+  @SerialVersionUID(1L) final case class Spawned(childName: String, props: Option[Props]) extends Effect
```

Comment:
Shall we, then, create a different `Effect` as well when calling `spawnAnonymous`?